src/pre_defined_classes/programs_with_changing_behaviour/medium/q11/MainTest.java

```diff
@@ -0,0 +1,42 @@
+package pre_defined_classes.programs_with_changing_behaviour.medium.q11;
+
+import global.BaseRandomTest;
+import global.variables.Clause;
+import global.variables.clauses.*;
+import org.junit.jupiter.api.RepeatedTest;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest extends BaseRandomTest {
+
```

Comment:
We also add a comment at the start to indicate what kind of question this is. If it is a java, we put `// Java`, if it is a Parsons question we put `// Parsons`, and if it is a Parsons with distractors we put `// Parsons with distractors`